app/routes.tsx

```diff
@@ -282,6 +283,7 @@ const RootRoutes: React.FC<RootRoutesProps> = props => {
       <DialogComponent />
       <FeedbackButton />
       <SnackbarComponent />
+      <PopupConsentBanner />
```

Comment:
I think that wrapping it with <NoSSR> will make the component safer.

Reply:
I don't understand `wapping it`. Is that mean,
```
<div className={s.wrapperStyle}>
  <PopupConsentBanner />
</div>
```
just this, or other mean?

Reply:
NoSSR. I guess Github system omited \<\> for some reason.